modules/drivers/sensors/sen0322/sen0322.js

```diff
@@ -0,0 +1,80 @@
+import Timer from "timer";
```

Comment:
Our ECMA-419 sensor drivers usually include a header with:

- Copyright notice
- Full name of part
- Link to relevant manufacturer data sheets
- Link to any implementations used as a reference